src/utils/client.js

```diff
@@ -0,0 +1,82 @@
+const jose = require('node-jose');
+const axios = require('axios');
+const { v4 } = require('uuid');
+const clientId = 'medmorph_backend';
+const keys = require('../keys/privateKey.json');
+
+class Client {
+  /**
+   *
+   * @param {*} jwks This clients private key set used for signing requests
+   */
+  constructor(jwks) {
+    this.jwks = jwks;
+  }
+
+  /**
+   * Generate and return access token for the specified server
+   * @param {string} url  the fhir base url for the server to connect to
+   */
+  async connectToServer(url) {
+    let response = await axios.get(`${url}/.well-known/smart-configuration`);
```

Comment:
This gives a lint warning about using const instead of let
```suggestion
    const response = await axios.get(`${url}/.well-known/smart-configuration`);
```